extras/lbryinc/client.go

```diff
@@ -13,11 +13,18 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-// Client stores data about internal-apis call it is about to make.
+// Caller interface defines currently supported internal-apis methods.
+type Caller interface {
+	SetServerAddress(string)
+	GetServerAddress() string
+	UserMe() (ResponseData, error)
+}
+
+// Client stores parameters common for internal-apis calls plus a logger.
 type Client struct {
-	ServerAddress string
+	serverAddress string
```

Comment:
What is the purpose of making these private and adding methods? There is no special logic in the getter/setter to require it.

Reply:
Can't define properties on an interface.